docker-compose.yml

```diff
@@ -16,5 +16,5 @@ services:
     env_file: docker.env
     environment:
       - SPEC_OPTS
-      #- JRUBY_OPTS
+      - LOGGER_LEVEL # devutils reads the ENV and sets LS logging
```

Comment:
would be easier to remember if we support [`LOG_LEVEL` env variable in devutils](https://github.com/elastic/logstash-devutils/blob/v2.0.3/lib/logstash/devutils/rspec/spec_helper.rb#L32).

useful for debugging LS behaviour on CI (used it heavily with the twitter integration tests run on Travis).

Reply:
will wait for https://github.com/elastic/logstash-devutils/pull/87 if we decide to rename ...